.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
Hi, Great job on adding the test.md file to the gitignore file 👍. Please also add node_modules to this gitignore file this ensure that the node_modules folder with all you packages doesn't get included in you repository. Thank you 🙏